anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -69,13 +86,18 @@ def _marshall_feed_response(feed: DbFeedMetadata) -> FeedMetadata:
     i.enabled = feed.enabled
     i.groups = []
 
-    for group in feed.groups:
+    groups = provider.get_feed_groups_detached(feed)
```

Comment:
so you're detaching this because FeedGroupMetadata backpopulates FeedMetadata?

Reply:
I detatch this for easier session management. The calling context only needs to read the attributes so they can be detatched as opposed to passing sessions around. I modeled this off the existing structure that used detatched records